src/misc.cpp

```diff
@@ -361,22 +361,7 @@ void std_aligned_free(void* ptr) {
 /// The returned pointer is the aligned one, while the mem argument is the one that needs
 /// to be passed to free. With c++17 some of this functionality could be simplified.
 
-#if defined(__linux__) && !defined(__ANDROID__)
-
-void* aligned_ttmem_alloc(size_t allocSize) {
-
-  void *mem = nullptr;
-  constexpr size_t alignment = 2 * 1024 * 1024; // assumed 2MB page sizes
-  size_t size = ((allocSize + alignment - 1) / alignment) * alignment; // multiple of alignment
-  if (posix_memalign(&mem, alignment, size))
-     mem = nullptr;
-#if defined(MADV_HUGEPAGE)
-  madvise(mem, size, MADV_HUGEPAGE);
-#endif
-  return mem;
-}
-
-#elif defined(_WIN64)
+#if defined(_WIN64)
```

Comment:
I think we could actually enable the VirtualAlloc and large pages path also on _WIN32. It compiles and works for me, at least, and even allocates with large pages.

I think the benefit here mainly would be to have one code path for Windows 32/64, and another for everything else. That might give a small perf boost for old win32 boxes.

Reply:
I you can test and it works, yes, I think that's a good thing.

Reply:
Ok, I'll push one more patch